get-started/code-experiments/README.md

```diff
@@ -0,0 +1,260 @@
+## DVC Get Started for Experiments
+
+This repository contains a project that trains a classifier for [Fashion
+MNIST][fmnist] dataset in Tensorflow. 
+
+[fmnist]: https://github.com/zalandoresearch/fashion-mnist
+
+It is used as a showcase for [`dvc exp`][gsdvcexp] commands to manage large
+number of experiments. 
+
+[gsdvcexp]: https://dvc.org/doc/start/experiments
+
+After [installing DVC][installdvc] and cloning the repository, you can run:
+
+[installdvc]: https://dvc.org/doc/install
+
+```console
+python -m venv .venv
+. .venv/bin/activate
+python -m pip install -r requirements.txt
+```
```

Comment:
We should add `dvc pull` here, otherwise next command (`dvc exp run...`) fails.

Reply:
Thank you @aguschin. 👍🏼  Fixed now.